module/validation/seal_validator.go

```diff
@@ -173,10 +173,8 @@ func (s *sealValidator) Validate(candidate *flow.Block) (*flow.Seal, error) {
 			// ATTENTION:
 			// Here, IncorporatedBlockID (the first argument) should be set
 			// to ancestorID, because that is the block that contains the
-			// ExecutionResult. However, in phase 2 of the sealing roadmap,
-			// we are still using a temporary sealing logic where the
-			// IncorporatedBlockID is expected to be the result's block ID.
-			incorporatedResults[resultID] = flow.NewIncorporatedResult(result.BlockID, result)
+			// ExecutionResult.
```

Comment:
I think we can remove this comment entirely
```golang
// ATTENTION:
// Here, IncorporatedBlockID (the first argument) should be set
// to ancestorID, because that is the block that contains the
// ExecutionResult.
```